libobs/graphics/graphics.c

```diff
@@ -947,8 +947,13 @@ static inline void assign_sprite_uv(float *start, float *end, bool flip)
 		*start = 0.0f;
 		*end = 1.0f;
 	} else {
+#if defined(_M_ARM64)
+		*start = 0.0f;
+		*end = 1.0f;
+#else
```

Comment:
Please elaborate why this change was made.

Reply:
During Functional testing, we noticed that the browser page was displayed upside down when compiled on ARM64. To address this, we adjusted the `start` and `end` values to ensure proper browser-page orientation in OBS Studio.

**Before Modification:**
![Before](https://github.com/user-attachments/assets/5702573a-fa08-46c5-a144-35da1a181cf5)

**After Modifications:**
![After](https://github.com/user-attachments/assets/4dbbdee9-103c-4a68-8948-13fb62428ab4)

Reply:
If that is the only purpose of this change, it can be removed. It's due to the newer shared texture API within CEF, and therefore is handled in https://github.com/obsproject/obs-browser/pull/434/files#diff-22b4408287e08499d62e82ed1c8b4e586d4dfe80c4ecdf18fb561c1027be627aL661-R661.